### DISCUSSION TITLE
flathub

### DISCUSSION THREAD

**Discussion by bztd** [wxMaxima-developers/wxmaxima#1835] [General]
Hello everyone, I don't know if this is the appropriate channel to discuss these topics, just to inform you that I have started a pull recuest so that wxMaxima is in the flathub repositories, I would like to know your opinion, if it is allowed or if I should give up.
This is the manifesto:
https://github.com/bztd/flathub/blob/io.gpl.wxmaxima/io.github.wxmaxima_developers.wxMaxima.yml
and this is the compilation for testing:
flatpak install --user https://dl.flathub.org/build-repo/61931/io.github.wxmaxima_developers.wxMaxima.flatpakref

**Comment by gunterkoenigsmann** [comment]
I never managed to build a flatpak out of wxMaxima...

**Reply by bztd** [reply]
So there is no problem with that?

**Reply by gunterkoenigsmann** [reply]
Q flatpak is something that has been requested multiple times and I never managed to make one. Thanks a lot!

**Comment by gunterkoenigsmann** [comment]
Is there a flatpak for Maxima, too? And do they both work together? That wasn't easy to archive for the snap package...

**Reply by bztd** [reply]
no. everything is together, gnuplot, maxima, sbcl, wxwidgets, wxMaxima.
I had problems with the upstream...
There is a drawback that bothered me a little with gnuplot and that is that I couldn't compile it with qt.
For the rest I have been using it and I have had no problems.

**Comment by bztd** [comment]
this is the url of the request
https://github.com/flathub/flathub/pull/4641
I am attentive to your suggestions
